Day-6-7/src/components/UserCard.js

```diff
@@ -1,19 +1,16 @@
 import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { fetchUserData } from "../utils/fetchData";
```

Comment:
Keep a blank line between global and local imports

Reply:
okay is it a convention?

Reply:
No, just a way to distinguish imports. You can order them in any way.